gulpfile.js

```diff
@@ -103,8 +103,11 @@ gulp.task('copy:main.css', function () {
 
     return gulp.src(dirs.src + '/css/main.css')
                .pipe(plugins.header(banner))
+               .pipe(plugins.autoprefixer({
+                    browsers: ['last 2 versions', 'ie >= 8'],
```

Comment:
@AllThingsSmitty Can you change this to:  `browsers: ['last 2 versions', 'ie >= 8', '> 1%']` ? Thanks.

--

@AllThingsSmitty Tip: You can use [`git commit --amend`](http://git-scm.com/book/en/v2/Git-Basics-Undoing-Things) and then [`git push -f origin <topic-branch-name>`](http://git-scm.com/docs/git-push) to avoid adding yet another commit to the pull request. :)